data-prepper-api/src/main/java/org/opensearch/dataprepper/model/annotations/DataPrepperPlugin.java

```diff
@@ -31,12 +31,21 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.TYPE})
 public @interface DataPrepperPlugin {
+    String DEFAULT_DEPRECATED_NAME = "unassigned";
```

Comment:
Maybe an empty string `""` or some really odd name would be better, like `"NO_DEPRECATED_NAME"`.

Reply:
I'm okay with `""`. I just used an example from documentation https://docs.oracle.com/javase/1.5.0/docs/guide/language/annotations.html